fixedwidth/fixedwidth.py

```diff
@@ -159,7 +159,7 @@ def validate(self):
             else: #no value passed in
 
                 #if required but not provided
-                if parameters['required'] and ('value' not in parameters):
+                if parameters['required'] and ('value' not in parameters) and self.data[field_name] is not None:
```

Comment:
If `None` is passed, the user will received a message saying that the field wasn't provided. I like the idea of disallowing `None`, so I think that should be a separate check with its own error message.

Maybe it would be better above, before line 138 -- something like:

```python
for k, v in self.data.items():
    if v is None:
        raise ValueError("None value not allowed for {0}".format(k))
```

Reply:
@ShawnMilo i added a custom check at line 174 and also changedthat None maps to an empty string when the line is generated.